instrumentation/faraday/lib/opentelemetry/instrumentation/faraday/version.rb

```diff
@@ -7,7 +7,7 @@
 module OpenTelemetry
   module Instrumentation
     module Faraday
-      VERSION = '0.19.3'
+      VERSION = '0.20.0'
```

Comment:
a minor bump was intentional here right?

Reply:
Yeah, it's actually automated.  Because `FEAT:` was used, it did a minor bump.

https://github.com/open-telemetry/opentelemetry-ruby/pull/1118/files#diff-6b8108d2c50d3daeb53032e989956edd30a1c181b60b57130b1030b5e75e2ba8R5-R6

Reply:
All the other ones are just fixes, so it was a patch bump for them.

Reply:
Gracias Amigo